apps/coscrad-frontend/src/configurable-front-matter/validation/validateFrontMatter.ts

```diff
@@ -0,0 +1,38 @@
+import { isStringWithNonzeroLength } from '@coscrad/validation';
+import FrontMatter from '../frontMatterData/FrontMatter';
+
+export const isErrorArray = (input: unknown): input is Error[] =>
+    Array.isArray(input) && input.every((item) => item instanceof Error);
+
+export default (input: unknown): FrontMatter | Error[] => {
```

Comment:
I actually kind of like this API. We went with `Error[]` only as the return type on Justin's latest PR. Let's make sure we are all on the same page.

Reply:
This has changed.